java/flight/flight-core/src/main/java/org/apache/arrow/flight/example/integration/AuthBasicProtoScenario.java

```diff
@@ -58,40 +58,37 @@ public void doAction(CallContext context, Action action, StreamListener<Result>
   public void buildServer(FlightServer.Builder builder) {
     builder.authHandler(new BasicServerAuthHandler(new BasicServerAuthHandler.BasicAuthValidator() {
       @Override
-      public byte[] getToken(String username, String password) throws Exception {
+      public Optional<String> validateCredentials(String username, String password) throws Exception {
+        if (Strings.isNullOrEmpty(username)) {
+          throw CallStatus.UNAUTHORIZED.withDescription("Credentials not supplied").toRuntimeException();
```

Comment:
Unauthorized != Unauthenticated, this should be Unauthenticated. The same goes on like 67 below.